utils/src/main/kotlin/Utils.kt

```diff
@@ -233,9 +233,13 @@ fun File.safeDeleteRecursively() {
  * @throws IOException if any missing directory could not be created.
  */
 fun File.safeMkdirs() {
-    if (this.isDirectory || this.mkdirs()) {
-        return
-    }
+    if (this.isDirectory) return
```

Comment:
How about keeping this as one-liner like `if (this.isDirectory || this.mkdirs() || this.isDirectory) return` and adjust the comment a bit to emphasize that we only distrust `mkdirs()` if it returns `false` by saying

> Do not blindly trust mkdirs() returning "false" as it can fail for edge-cases like
> File(File("/tmp/parent1/parent2"), "/").mkdirs()
> although the directory is successfully created.

As an added benefit of this, we only do the second call to `isDirectory` if `mkdirs()` returned `false`.

Reply:
Done.